core/src/main/java/org/infinispan/protostream/ProtobufUtil.java

```diff
@@ -154,7 +154,7 @@ public static ByteBuffer toWrappedByteBuffer(ImmutableSerializationContext ctx,
    }
 
    public static void toWrappedStream(ImmutableSerializationContext ctx, OutputStream out, Object t) throws IOException {
-      toWrappedStream(ctx, out, t, DEFAULT_STREAM_BUFFER_SIZE);
+      WrappedMessage.write(ctx, TagWriterImpl.newInstance(ctx, out), t);
```

Comment:
shouldn't be `TagWriterImpl.newInstanceNoBuffer`?

Reply:
Ah! I see you changed `newInstance(ImmutableSerializationContext serCtx, OutputStream output)`. 
So `newInstanceNoBuffer` is kind of duplicated/irrelevant at this point!?

Reply:
I was not aware of newInstanceNoBuffer! Yeah, no reason newInstance should ever use the buffered version imo. Updated to deprecate newInstanceNoBuffer as well.